src/dmd/parse.d

```diff
@@ -4253,7 +4255,8 @@ final class Parser(AST) : Lexer
     }
 
     private void parseStorageClasses(ref StorageClass storage_class, ref LINK link,
-        ref bool setAlignment, ref AST.Expression ealign, ref AST.Expressions* udas)
+        ref bool setAlignment, ref AST.Expression ealign, ref AST.Expressions* udas,
+        ref Loc linkloc)
```

Comment:
Should this be `out`, or is it used for input as well?

Reply:
You're right. Changed it from `ref` to `out`